MdeModulePkg/Core/PiSmmCore/PiSmmIpl.c

```diff
@@ -554,13 +554,11 @@ SmmCommunicationCommunicate (
   IN OUT UINTN                             *CommSize OPTIONAL
   )
 {

```

Comment:
I know we've discussed this offline, but the commit message and/or PR description should link to more details about the issue.

Reply:
PR description updated.